.github/workflows/ci-build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install Rust toolchain
-      uses: actions-rs/toolchain@v1
+      uses: dtolnay/rust-toolchain@v1
```

Comment:
The second and third invocations should use the `stable` branch, as the `v1` tag is stale.

Reply:
I also forgot to strip off some obsolete arguments that aren't used by `dtolnay/rust-toolchain@v1`.